sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,15 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        unit = "DN"
```

Comment:
Are we sure we can set this to DN?

Reply:
that was my understanding of the issue but I'm not sure. I would like @wtbarnes to verify. I tried going through the XRT analysis guide, and I think I'm missing accounting for level 2 data?

Reply:
Where is this guide?

Reply:
it was in the original issue (#6716) description, here's the [link](https://xrt.cfa.harvard.edu/resources/documents/XAG/XAG.pdf), table 1.1. Will also described things pretty well in the issue description.

Reply:
I see it, I think we would check for the unit in the metadata, and if non, set it to DN, and also check the history tag to see if it should be DN/S. 

A reference to that document and the table number would be good here as well. If it isn't already in the main doc string.

Reply:
I had a better look at the document and Will's responses in the original issue, and i think it'd be better to mention section 2.11 of the XRT Analysis Guide, I'll do that :+1: